src/common.cc

```diff
@@ -163,4 +163,39 @@ DecodeBase64(
   return nullptr;
 }
 
+TRITONSERVER_Error*
+ValidateSharedMemoryKey(const std::string& name, const std::string& shm_key)
+{
+  std::string_view key_view(shm_key);
+
+  // Find the index of the first character that is not a slash
+  const std::size_t first_non_slash = key_view.find_first_not_of('/');
+
+  // If the entire key is slashes
+  if (first_non_slash == std::string_view::npos) {
```

Comment:
This validation is optional; if we skip it, the code will fail later.
Since we need to check for slashes in this function, I added this validation to return error early.